Things.md

```diff
@@ -169,4 +170,10 @@ Events can also be added to repeat, similar to using `setInterval`.  A new third
 FSM.addEventInterval(FSM.shiftHoriz, 100, 5, thing, 2);
 ```
 
-More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
\ No newline at end of file
+More can be read about TimeHandlr on its [Readme](https://github.com/FullScreenShenanigans/TimeHandlr/blob/master/README.md).
+
+# Thing classes
+
+Classes are implemented using [ObjectMakr](https://github.com/FullScreenShenanigans/ObjectMakr), a module that automates the creation of the inheritance structure of classes. The structure and a list of properties are given and the classes can then be constructed. 
+
+In respect to FullScreenShenanigans, the inheritance structure and properties are specified in an `objects.js` file. When a new Thing is created, ObjectMakr references its information in `objects.js` and constructs the new object. Most of the basic properties, like `height`, `width`, and `opacity` are defined in the general Thing class. Each subclass (Block, tree, etc.) has the option to use, update, or add on to these properties.
```

Comment:
You can probably get rid of the last line. When you clarify in earlier code that they're just regular classes, it will become redundant.